src/js/models/results/award/AwardSummary.js

```diff
@@ -12,63 +12,141 @@ import GenericRecord from '../GenericRecord';
+    'funding_office_name',
+    'recipient_street',
+    'recipient_city',
+    'recipient_state_province',
+    'recipient_zip_postal',
+    'recipient_country'
 ];
 
 const remapData = (data, idField) => {
     // remap expected child fields to top-level fields
     const remappedData = data;
+    let id = '';
+    let awardType = '';
+    let awardTypeDescription = '';
     let awardingAgencyName = '';
     let awardingSubtierName = '';
+    let awardingOfficeName = '';
     let fundingAgencyName = '';
     let fundingSubtierName = '';
+    let fundingOfficeName = '';
     let recipientName = '';
+    let recipientStreet = '';
+    let recipientCity = '';
+    let recipientStateProvince = '';
+    let recipientZipPostal = '';
+    let recipientCountry = '';
+
+    if (data.id) {
+        id = data.id;
+    }
+
+    if (data.type) {
+        awardType = data.type;
+    }
+
+    if (data.type_description) {
+        awardTypeDescription = data.type_description;
+    }
 
     if (data.awarding_agency) {
         awardingAgencyName = data.awarding_agency.toptier_agency.name;
         awardingSubtierName = data.awarding_agency.subtier_agency.name;
+        if (data.awarding_agency.office_agency) {
+            awardingOfficeName = data.awarding_agency.office_agency.name;
+        }
     }
 
     if (data.funding_agency) {
         fundingAgencyName = data.funding_agency.toptier_agency.name;
         fundingSubtierName = data.funding_agency.subtier_agency.name;
+        if (data.funding_agency.office_agency) {
+            fundingOfficeName = data.funding_agency.office_agency.name;
+        }
     }
 
-    if (data.recipient) {
-        recipientName = data.recipient.recipient_name;
-    }
+    remappedData.type = awardType;
+    remappedData.type_description = awardTypeDescription;
     remappedData.awarding_agency_name = awardingAgencyName;
     remappedData.awarding_subtier_name = awardingSubtierName;
+    remappedData.awarding_office_name = awardingOfficeName;
     remappedData.funding_agency_name = fundingAgencyName;
     remappedData.funding_subtier_name = fundingSubtierName;
+    remappedData.funding_office_name = fundingOfficeName;
     remappedData.recipient_name = recipientName;
+    remappedData.id = id;
 
-    // set the ID to the relevant field
-    let id = data.fain;
+    // set the awardID (fain or piid) to the relevant field
+    let awardId = data.fain;
     if (!idField) {
         // unspecified ID field, use whatever value is available
         if (!data.fain && data.piid) {
-            id = data.piid;
+            awardId = data.piid;
         }
         else if (data.uri) {
-            id = data.uri;
+            awardId = data.uri;
         }
     }
     else {
-        id = data[idField];
-        if (!id) {
-            id = '';
+        awardId = data[idField];
+        if (!awardId) {
+            awardId = '';
         }
     }
-    remappedData.id = id;
+    remappedData.awardId = awardId;
```

Comment:
Updating the `awardId` property to `award_id` to be in line with the rest of the values as well.